### ISSUE TITLE
Display two entries with same priority does not show simultaneously as intended

### ISSUE THREAD

**Issue by lingxiao** [huri000/SwiftEntryKit#293]
**Describe the bug**
I have a `.topFloat`  and `bottomNote` entry, both set to `.normal` with:
`    attributes.precedence = .override(priority: .normal, dropEnqueuedEntries: false) `,

when display both with `SwiftEntryKit.display(entry: modal, using: attributes)`, the second entry is flushed from the queue and does not display as intended. 

**To Reproduce**

**Expected behavior**

Both entries should appear on screen at the same time.

**Screenshots**

**iPhone (please complete the following information):**
 - Device: [e.g. iPhone6]    iphone X
 - iOS Version: [e.g. iOS9.3]  13.5.1
 - Xcode Version [e.g. 9.3] 11.5
 - Dependency Manager Version [e.g. CocoaPods 1.5.2, Carthage 0.29.0]  Pod
 - SwiftEntryKit Release # [e.g. 0.2.0]  1.1.4

**Additional context**

**Screenshots / Video Links**

**Comment by huri000** [comment]
@lingxiao what you described is the intended behavior. Entries cannot be displayed simultaneously by design (at least not with that library). If you want to show a view2 you'll have to dismiss view1 - that priority ratio required is `>=`, so if both views' priorities are equal the second view will still dismiss the first.

TL;DR: only one entry is permitted as a time.
